### ISSUE TITLE
Installation warning

### ISSUE THREAD

**Issue by r3d9u11** [filcuc/nimqml#13]
How. This library is active and will support in future? I have notice in installation:
```
$nimble install nimqml
Downloading https://github.com/filcuc/nimqml using git
   Warning: Package 'nimqml' has an incorrect structure. It should contain a single directory hierarchy for source files, named 'nimqml', but file 'qguiapplication.nim' is in a directory named 'private' instead. This will be an error in the future.
      Hint: If 'private' contains source files for building 'nimqml', rename it to 'nimqml'. Otherwise, prevent its installation by adding `skipDirs = @["private"]` to the .nimble file.
```

**Comment by filcuc** [comment]
Yes i will fix. The actual project structure was the correct structure in
previous nimble versions. The convention changes in last nimble releases.
That said it's just a warning I don't think it blocks you

Il 26 Feb 2018 7:58 PM, "r3d9u11" <notifications@github.com> ha scritto:

**Comment by r3d9u11** [comment]
@filcuc It is not blocked me, not yet. But "This will be an error in the future.", so I'm asking about future of this library :)

**Comment by r3d9u11** [comment]
@filcuc also, is it possible to use non-standard path to Qt? (I have installed it to /home/username/Qt via online-installer qt-unified-linux-x64-3.0.2-online.run)

**Comment by filcuc** [comment]
Fixed in 61d2cd0f93d20960e0a259b79c4e0ec0f4a3d648
